dali/base/dacoven.cpp

```diff
@@ -321,6 +321,12 @@ class CCovenBase: implements ICoven, public CInterface
         if (comm)
             comm->cancel(srcrank,tag);
     }
+    virtual const SocketEndpoint &queryChannelPeerEndpoint(const SocketEndpoint &sender) override
+    {
+        if (!comm)
```

Comment:
assertex(comm) would be clearer, perhaps (and give a marginally more helpful error if triggered).

Reply:
Will change this and a similar one in mpcomm.cpp's queryChannelPeerEndpoint() to an assertex